demisto_sdk/scripts/init_validation_script.py

```diff
@@ -13,6 +13,23 @@
     "D": "GitStatuses.DELETED",
 }
 
+RELATED_FILES_DICT = {
+    "1": "RelatedFileType.YML",
+    "2": "RelatedFileType.JSON",
+    "3": "RelatedFileType.README",
+    "4": "RelatedFileType.DESCRIPTION",
+    "5": "RelatedFileType.IMAGE",
+    "6": "RelatedFileType.DARK_SVG",
+    "7": "RelatedFileType.LIGHT_SVG",
+    "8": "RelatedFileType.CODE",
+    "9": "RelatedFileType.TEST_CODE",
+    "10": "RelatedFileType.SCHEMA",
+    "11": "RelatedFileType.XIF",
+    "12": "RelatedFileType.PACK_IGNORE",
+    "13": "RelatedFileType.SECRETS_IGNORE",
+    "14": "RelatedFileType.AUTHOR_IMAGE",
+}
```

Comment:
why do we need the numbers as keys?

Reply:
To allow however use this script to easily use it for adding related_files section with the numbers rather than text.